src/update-cb.hh

```diff
@@ -56,14 +56,15 @@ namespace conky {
 
 			semaphore sem_start;
 			std::thread *thread;
-			const size_t hash;
-			uint32_t period;
-			uint32_t remaining;
+			const size_t hash;		// used to determined callback uniqueness
```

Comment:
Ah, good to know about the `/* */` comment blocks! I've seen both, so I wasn't sure.